vueui/src/components/view/devices/gps/gps.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <!-- Controls -->
+    <h3> {{ own_tag }} </h3>
+    <vui-button :params="{ tag: 1 }" width="3em">Set GPS Tag</vui-button>
+    <vui-button :params="{ add_tag: 1 }" width="3em">Track New Tag</vui-button>
+    <vui-button :params="{ add_all: 1 }" width="3em">Track All</vui-button>
+    <vui-button :params="{ clear_all: 1 }" width="3em">Untrack All</vui-button>
+    <hr>
+
+    <!-- Tracking List -->
+    <div class="table">
+      <div class="header">
+        <div class="header-item">GPS Tag</div>
+        <div class="header-item">Location</div>
+        <div class="header-item">Area</div>
+        <div class="header-item">Remove</div>
+      </div>
+      <div class="header" v-for="gps in tracking_list" :key="gps.tag">
+        <div class="item"> {{ gps.tag }} </div>
+        <div class="item center"> {{ gps.pos_x }}, {{ gps.pos_y }}, {{ gps.pos_z }} </div>
+        <div class="item left"> {{ gps.area }} </div>
+        <vui-button v-if="gps.tag != own_tag" class="item center" :params="{ remove_tag: gps.tag }" width="3em">Untrack</vui-button>
+        <div v-else class="item center"/>
+      </div>
+    </div>
+</div></template>
+
+<script>
+export default {
+  data() {
+    return this.$root.$data.state;
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.table {
+  display: table;
+  width: 100%;
+
+  .header {
+    display: table-row;
+    .header-item, {
+      font-weight: bold;
+      display: table-cell;
+      padding: 1px;
+      vertical-align: middle;
+    }
+    .item {
+      font-weight: normal;
+      display: table-cell;
+      padding: 1px;
+      vertical-align: middle;
+    }
+  }
+}
+
+.center {
+  text-align: center;
+}
+
+.right {
+  text-align: right;
+}
```

Comment:
Look at https://github.com/Aurorastation/Aurora.3/pull/8057#discussion_r370924012 for guidance